assets/vega_lite/main.js

```diff
@@ -120,15 +120,17 @@ export function init(ctx, data) {
     "https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap"
   );
 
-  const { spec, datasets } = data;
+  const { spec, datasets, config } = data;
```

Comment:
I pass a config object instead of just the theme, in case we want to configure more things in the future